commands/bake.go

```diff
@@ -23,9 +24,20 @@ func runBake(dockerCli command.Cli, targets []string, in bakeOptions) error {
 	ctx := appcontext.Context()
 
 	if len(in.files) == 0 {
-		files, err := defaultFiles()
-		if err != nil {
-			return err
+		var files []string
+		composeFileVar, ok := os.LookupEnv("COMPOSE_FILE")
+		if ok {
+			composePathSeparator, ok := os.LookupEnv("COMPOSE_PATH_SEPARATOR")
```

Comment:
Hm, don't think this one is currently supported by `docker stack deploy` 🤔

Reply:
I think that counts for both of them, to be honest. Being able to avoid installing `docker-compose`, but rather use `docker buildx bake` and `docker stack deploy` while being able to read `COMPOSE_FILE` and even perhaps a `.env` file would be golden.